frontend/src/Pages/LoginSuccess.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useSearchParams } from 'react-router-dom'
+import axios from 'axios';
+
+export default function LoginSuccess() {
+    const [searchParams, setSearchParams] = useSearchParams();
+    const loginToken = searchParams.get('loginToken');
```

Comment:
Would you mind expanding how the flow is meant to work at a high level in your PR description? I'm assuming this is a callback page which transforms a login token into a fully fledged access token, but this needs more documentation.

Reply:
Done.